### PR TITLE
Updated piman_README.md to include documentation about mapper()

### PR DESCRIPTION
**Pull request by StevenvL**
https://sjsu.instructure.com/courses/1375041/discussion_topics/4088277

### CODE REVIEW

**Comment by YehyaHadi** [comment]
Looks good to me, gj!

**Review by YehyaHadi — APPROVED** [review]
Looks great, gj!

**Review by spencer-enriquez — APPROVED** [review]
Also looks good!

File: piman_README.md [hunk 1]
```diff
@@ -87,7 +87,48 @@ The reinstallation process uses `/utility/power_cycle.py` to turn the pi at the
...
+    time.sleep(30)
+    mac_mapper.mac_mapper()
+```
+`mapper()` is a function inside the `piman.py` file. It ultimately encompasses two functions `power_cycle.power_cycle()` and `mac_mapper.mac_mapper()` which will power cycle the pis causing them to try PXE boot and afterwards getting the mac address of each specified pi. This output is then printed to console as well as `mac_mapper.txt`
+
+**How to use mapper()**
+
+The official implementation is specified as `mapper(switch_address,interface, port)`.
+
+`switch_address` (String) is your specified ip ending with .129.
```
Comment (breed) [inline piman_README.md]: you should say the IP address of the switch here since sometimes it will have a different ending.

File: piman_README.md [hunk 2]
```diff
@@ -87,7 +87,48 @@ The reinstallation process uses `/utility/power_cycle.py` to turn the pi at the
...
+
+`interface` (Int) use `ifconfig` to figure out the interface that the pis are on
+
+`port` (int[]) is the specified remote pi that you want to access, ranging from either ports 1-10 or 11-20 depending on your team number.
```
Comment (breed) [inline piman_README.md]: team is a class concept, not a general one, so you should just stick to the port number

File: piman_README.md [hunk 3]
```diff
@@ -87,7 +87,48 @@ The reinstallation process uses `/utility/power_cycle.py` to turn the pi at the
...
+```
+`mapper()` is a function inside the `piman.py` file. It ultimately encompasses two functions `power_cycle.power_cycle()` and `mac_mapper.mac_mapper()` which will power cycle the pis causing them to try PXE boot and afterwards getting the mac address of each specified pi. This output is then printed to console as well as `mac_mapper.txt`
+
+**How to use mapper()**
+
+The official implementation is specified as `mapper(switch_address,interface, port)`.
+
+`switch_address` (String) is your specified ip ending with .129.
+
+`interface` (Int) use `ifconfig` to figure out the interface that the pis are on
```
Comment (breed) [inline piman_README.md]: isn't this usually one. i'm not sure how you get that with ifconfig

**Review by breed — COMMENTED** [review]
just a few things to fix. also please remove the phase and the city name from the title :)

**Review by breed — APPROVED** [review]
looks great! thanx!

**Pull request merged**